sqlmesh/core/model/meta.py

```diff
@@ -62,6 +62,7 @@ class ModelMeta(PydanticModel):
     column_descriptions_: t.Optional[t.Dict[str, str]]
     audits: t.List[AuditReference] = []
     tags: t.List[str] = []
+    hash_raw_query: bool = Field(default=False)
```

Comment:
why do you need field? why can't you make it bool = False?

Reply:
I thought Python 3.8 / 3.9 failures were my fault so I tried to address them. Will remove now